theories/_CoqProject

```diff
@@ -225,6 +225,9 @@ FOL/binFOL.v
 FOL/binFOL_undec.v
 FOL/PA.v
 FOL/PA_undec.v
+FOL/FSAT.v
```

Comment:
Do we want to mention those files instead of the `TRAKHTENBROT` directory in the README?

Reply:
Perhaps both?

Reply:
Well maybe `FSAT` as an entry point and a comment linking to `TRAKHTENBROT` for the detailed implementation because `FOL/Reductions/TRAKHTENBROT_to_FSAT.v` is just glue code translating from one syntax to the other.

Reply:
Okay, will do.